libethdrivers/src/virtio_pci.c

```diff
@@ -262,9 +262,9 @@ static void fill_rx_bufs(struct eth_driver *driver) {
             .next = 0
         };
         dev->rx_ring.avail->ring[dev->rx_ring.avail->idx % dev->rx_size] = dev->rdt;
-        asm volatile("sfence" ::: "memory");
+        __atomic_thread_fence(__ATOMIC_RELEASE);
         dev->rx_ring.avail->idx++;
-        asm volatile("sfence" ::: "memory");
+        __atomic_thread_fence(__ATOMIC_RELEASE);
```

Comment:
This maps to `asm volatile ("" ::: "memory")` which I believe should still be correct due to x86's memory semantics

Reply:
tested it?

Reply:
yea